qiskit_aqt_provider/aqt_provider.py

```diff
@@ -314,3 +317,12 @@ def get_backend(
             raise QiskitBackendNotFoundError("No backend matches the criteria")
 
         return backends[0]
+
+    def get_direct_access_backend(self, *, host: str, port: int) -> AQTDirectAccessResource:
```

Comment:
Wouldn't a URL parameter not be easier? This also would support the case when the API is routed to a different prefix.

The `AQTDirectAccesResource` also supports `https` urls, but this is not exposed here.

Reply:
Changed in ef8aad5.